### PR TITLE
ticdc: add how to use base64 encoded mysql password

### PR DESCRIPTION
**Pull request by asddongmen**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.3 (TiDB 6.3 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: ticdc/ticdc-sink-to-mysql.md [hunk 1]
```diff
@@ -58,10 +58,17 @@ URI 中可配置的参数如下：
...
+若需要对 Sink URI 中的数据库密码使用 base64 进行编码，可以参考如下操作:
+
+```shell
+$ echo -n '123456' | base64
+MTIzNDU2
+```
 
 > **注意：**
-> URI 中包含特殊字符时，如 `! * ' ( ) ; : @ & = + $ , / ? % # [ ]`，需要对 URI 特殊字符进行转义处理。你可以在 [URI Encoder](https://meyerweb.com/eric/tools/dencoder/) 中对 URI 进行转义。
+> 当 Sink URI 中包含特殊字符时，如 `! * ' ( ) ; : @ & = + $ , / ? % # [ ]`，需要对 URI 特殊字符进行转义处理。你可以在 [URI Encoder](https://meyerweb.com/eric/tools/dencoder/) 中对 URI 进行转义。
```
Comment (shichun-0415) [inline ticdc/ticdc-sink-to-mysql.md:71]:
```suggestion
>
> 当 Sink URI 中包含特殊字符时，如 `! * ' ( ) ; : @ & = + $ , / ? % # [ ]`，需要对 URI 特殊字符进行转义处理。你可以在 [URI Encoder](https://meyerweb.com/eric/tools/dencoder/) 中对 URI 进行转义。
```

**Review by IANTHEREAL — APPROVED** [review]

File: ticdc/ticdc-sink-to-mysql.md [hunk 2]
```diff
@@ -49,7 +49,7 @@ URI 中可配置的参数如下：
 | 参数         | 描述                                             |
 | :------------ | :------------------------------------------------ |
 | `root`        | 下游数据库的用户名。                             |
-| `123456`       | 下游数据库密码。                                     |
+| `123456`       | 下游数据库密码。（可采用 base64 进行编码）                                     |
```
Comment (TomShawn) [inline ticdc/ticdc-sink-to-mysql.md:52]:
```suggestion
| `123456`       | 下游数据库密码。（可采用 Base64 进行编码）                                     |
```

File: ticdc/ticdc-sink-to-mysql.md [hunk 3]
```diff
@@ -58,10 +58,18 @@ URI 中可配置的参数如下：
 | `ssl-cert`     | 连接下游 MySQL 实例所需的证书文件路径（可选）。 |
 | `ssl-key`      | 连接下游 MySQL 实例所需的证书密钥文件路径（可选）。 |
 | `time-zone`    | 连接下游 MySQL 实例时使用的时区名称，从 v4.0.8 开始生效。（可选。如果不指定该参数，使用 TiCDC 服务进程的时区；如果指定该参数但使用空值，则表示连接 MySQL 时不指定时区，使用下游默认时区）。 |
-| `transaction-atomicity`      | 指定事务的原子性级别（可选，默认值为 `table`）。当该值为 `table` 时 TiCDC 保证单表事务的原子性，当该值为 `none` 时 TiCDC 会拆分单表事务。 |
+| `transaction-atomicity`      | 指定事务的原子性级别（可选，默认值为 `none`）。当该值为 `table` 时 TiCDC 保证单表事务的原子性，当该值为 `none` 时 TiCDC 会拆分单表事务。 |
+
+若需要对 Sink URI 中的数据库密码使用 base64 进行编码，可以参考如下操作:
```
Comment (TomShawn) [inline ticdc/ticdc-sink-to-mysql.md:63]:
```suggestion
若需要对 Sink URI 中的数据库密码使用 Base64 进行编码，可以参考如下操作。假设密码为 `123456`，使用如下命令可得到加密后的密码为 `MTIzNDU2`。
```

File: ticdc/ticdc-sink-to-mysql.md [hunk 4]
```diff
@@ -58,10 +58,18 @@ URI 中可配置的参数如下：
...
+
+```shell
+$ echo -n '123456' | base64
```
Comment (TomShawn) [inline ticdc/ticdc-sink-to-mysql.md:66]:
```suggestion
echo -n '123456' | base64
```

File: ticdc/ticdc-sink-to-mysql.md [hunk 5]
```diff
@@ -58,10 +58,18 @@ URI 中可配置的参数如下：
...
+MTIzNDU2
```
Comment (TomShawn) [inline ticdc/ticdc-sink-to-mysql.md:67]:
```suggestion
MTIzNDU2   # 加密后的密码
```

File: ticdc/ticdc-sink-to-mysql.md (hunk 3 above)
Comment (TomShawn) [inline ticdc/ticdc-sink-to-mysql.md:63]:
```suggestion
若需要对 Sink URI 中的数据库密码使用 Base64 进行编码，可以参考如下操作:
```

**Review by TomShawn — APPROVED** [review]
rest LGTM

File: ticdc/ticdc-sink-to-mysql.md [hunk 6]
```diff
@@ -58,10 +58,18 @@ URI 中可配置的参数如下：
...
 | `time-zone`    | 连接下游 MySQL 实例时使用的时区名称，从 v4.0.8 开始生效。（可选。如果不指定该参数，使用 TiCDC 服务进程的时区；如果指定该参数但使用空值，则表示连接 MySQL 时不指定时区，使用下游默认时区）。 |
-| `transaction-atomicity`      | 指定事务的原子性级别（可选，默认值为 `table`）。当该值为 `table` 时 TiCDC 保证单表事务的原子性，当该值为 `none` 时 TiCDC 会拆分单表事务。 |
+| `transaction-atomicity`      | 指定事务的原子性级别（可选，默认值为 `none`）。当该值为 `table` 时 TiCDC 保证单表事务的原子性，当该值为 `none` 时 TiCDC 会拆分单表事务。 |
+
+若需要对 Sink URI 中的数据库密码使用 Base64 进行编码，可以参考如下操作：
+
+```shell
+echo -n '123456' | base64   # 假设加密前的密码为 123456
+MTIzNDU2   # 加密后的密码
+```
```
Comment (TomShawn) [inline ticdc/ticdc-sink-to-mysql.md:68]:
````suggestion
若需要对 Sink URI 中的数据库密码使用 Base64 进行编码，可以参考如下命令：

```shell
echo -n '123456' | base64   # 假设待加密的密码为 123456
```

加密后的密码如下：

```shell
MTIzNDU2
```
````

File: ticdc/ticdc-changefeed-config.md [hunk 7]
```diff
@@ -51,23 +51,27 @@ case-sensitive = true
...
+# sync-point-interval = "5m"
 
 # Syncpoint 功能在下游表中保存的数据的时长，超过这个时间的数据会被清理
 # 配置格式为 h m s，例如 "24h30m30s"
 # 默认值为 24h
-sync-point-retention = "1h"
+# sync-point-retention = "1h"
+
+[mounter]
+# mounter 解码 kv 数据的线程数，默认值为 16
```
Comment (Oreoxmt) [inline ticdc/ticdc-changefeed-config.md:69]:
```suggestion
# mounter 解码 KV 数据的线程数，默认值为 16
```

File: ticdc/ticdc-sink-to-mysql.md [hunk 8]
```diff
@@ -58,10 +58,18 @@ URI 中可配置的参数如下：
...
 
 > **注意：**
-> URI 中包含特殊字符时，如 `! * ' ( ) ; : @ & = + $ , / ? % # [ ]`，需要对 URI 特殊字符进行转义处理。你可以在 [URI Encoder](https://meyerweb.com/eric/tools/dencoder/) 中对 URI 进行转义。
+>
+> 当 Sink URI 中包含特殊字符时，如 `! * ' ( ) ; : @ & = + $ , / ? % # [ ]`，需要对 URI 特殊字符进行转义处理。你可以在 [URI Encoder](https://meyerweb.com/eric/tools/dencoder/) 中对 URI 进行转义。
```
Comment (Oreoxmt) [inline ticdc/ticdc-sink-to-mysql.md:72]:
```suggestion
> 当 Sink URI 中包含特殊字符时，如 `! * ' ( ) ; : @ & = + $ , / ? % # [ ]`，需要对 URI 特殊字符进行转义处理。你可以使用 [URI Encoder](https://meyerweb.com/eric/tools/dencoder/) 工具对 URI 进行转义。
```

**Review by Oreoxmt — APPROVED** [review]
Rest LGTM

File: ticdc/ticdc-sink-to-mysql.md (hunk 6 above)
Comment (Oreoxmt) [inline ticdc/ticdc-sink-to-mysql.md:68]:
````suggestion
若需要对 Sink URI 中的数据库密码使用 Base64 进行编码，可以参考如下命令：

```shell
echo -n '123456' | base64   # 假设待编码的密码为 123456
```

编码后的密码如下：

```shell
MTIzNDU2
```
````

**Pull request merged**